web/dashboard/src/styles/themes.css

```
@@ -0,0 +1,181 @@
+  --cal-level-10: color-mix(in srgb, var(--info) 62%, #ffffff);
+  --alias-row-valid-bg: color-mix(
+    in srgb,
+    var(--bg-surface-hover) 86%,
+    #fff 14%
+  );
+  --alias-row-valid-bg-hover: color-mix(
+    in srgb,
+    var(--bg-surface-hover) 72%,
+    #fff 28%
+  );
+  color-scheme: dark;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)